doc/manual/build.sh

```diff
@@ -5,6 +5,9 @@ if [ $GEM_SET_DEBUG ]; then
 fi
 set -e
 
+version="$(cat openquake/baselib/__init__.py | sed -n "s/^__version__[  ]*=[    ]*['\"]\([^'\"]\+\)['\"].*/\1/gp")"
```

Comment:
This would make this script work only if called from the project root, otherwise this path would not be found.

I would: save actual path of the script and call `$CURPATH/../../openquake/baselib/__init__.py` this will make the script 'portable'.

Reply:
You can use something like 
`$CURPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`

to get the dir where the script is.

Reply:
Thanks @daniviga, done!